### PR TITLE
Bookstore: use Redux in React components.

### PR DESCRIPTION
**Pull request by harshitajain06**
- Understood the concept of state management.
- Used store, actions and reducers in React.
- Connected React and Redux.

### CODE REVIEW

**Comment by DammyShittu** [comment]
Hi @harshitajain06,

While you made a great effort in this project, I cannot proceed to review your code.

## Invalid Code Review Request

### Kindly remove the microverse logo from your readme or add yours instead if you have one

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/pages/Category.js [hunk 1]
```diff
@@ -1,9 +1,17 @@
...
-);
+const Category = () => {
+  const dispatch = useDispatch();
+  const status = useSelector((state) => state.category);
+
+  return (
+    <div className="flex justify-center align-middle">
+      <button onClick={() => dispatch(checkStatus())} className="bg-blue-500 cursor-pointer mr-20 mt-14 mb-14 text-sm px-2 py-1 text-white font-bold" type="button">CHECK STATUS</button>
+      <h1>{status}</h1>
+    </div>
```
Comment (Favourezeugwa) [inline src/pages/Category.js]: - In your "src/redux/categories/categories.js", I will like to kindly suggest that when the user clicks on the button `Check status`, it returns the phrase `Under Construction`. 🌸

**Review by Favourezeugwa — CHANGES_REQUESTED** [review]
## STATUS: REQUIRES CHANGES 🚦 ⚠️ ♻️ 

Hi @harshitajain06 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

close lady

## Highlights 💯 
- No Linter errors ✔️
- Descriptive Pull request summary✔️
- Good commit messages ✔️
- Professional Readme file ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Davidosky007 — APPROVED** [review]
Hi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
* No linter errors 
* Proper use of Gitflow 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**